CMakeLists.txt

```diff
@@ -180,6 +180,17 @@ IF (EMSCRIPTEN)
   ADD_LINK_FLAG("-s DISABLE_EXCEPTION_CATCHING=0")
   # make the tools immediately usable on Node.js
   ADD_LINK_FLAG("-s NODERAWFS")
+  # this can be moved into the fastcomp section once upstream ignores this flag,
+  # https://github.com/emscripten-core/emscripten/pull/9897
+  ADD_COMPILE_FLAG("-Wno-almost-asm")
```

Comment:
I'm confused; the linked pull request has been merged, so is this still necessary?

Reply:
I'd prefer to leave it for now, as it lets people use older emscripten versions. For example, getting binaryen master and compiling it was one emscripten version back (from the one tagged this morning) or earlier would fail without this.